files/install-ptfe.sh

```diff
@@ -290,6 +292,20 @@ if [ "x${role}x" == "xmainx" ]; then
             "--airgap-installer ${replicated_installer_path}"
         )
     fi
+
+    #If a custom weave CIDR is provided, set the necessary arguement
+    if [[ $(< /etc/ptfe/weave-cidr) != none ]]; then
```

Comment:
since you set the default string for this to be empty https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/11/files#diff-04c6e90faac2675aa89e2176d2eec7d8R56

I don't think this will work as written, 

we could update it to be:
```suggestion
    if [[ $(< /etc/ptfe/weave-cidr) != "" ]]; then
```